internal/metastructure/datastore/migrations_sqlite/00002_add_discoverable_column.sql

```diff
@@ -0,0 +1,8 @@
+-- +goose Up
+ALTER TABLE targets ADD COLUMN discoverable INTEGER DEFAULT 0;
```

Comment:
This script fails for me because the column already exists. I believe the common pattern for this is catching and ignoring the error as SQLite doesn't have `IF NOT EXISTS` syntax for adding columns.

```
2025-11-01T12:26:11-07:00 ERR Failed to run migrations error="ERROR 00002_add_discoverable_column.sql: failed to run SQL migration: failed to execute SQL query \"ALTER TABLE targets ADD COLUMN discoverable INTEGER DEFAULT 0;\": duplicate column name: discoverable"
2025-11-01T12:26:11-07:00 ERR Failed to create ms error="ERROR 00002_add_discoverable_column.sql: failed to run SQL migration: failed to execute SQL query \"ALTER TABLE targets ADD COLUMN discoverable INTEGER DEFAULT 0;\": duplicate column name: discoverable"
```